<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamView</class>
 <widget class="QWidget" name="StreamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>243</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="stream_view">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cameraName">
     <property name="text">
      <string>Unknown Camera</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="removeStream">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseStream">
       <property name="text">
        <string>Pause Stream</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="location">
     <property name="text">
      <string>Unknown Location</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
